components/image/Image.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * Author: bladimirardiles
+ * Component File Name: Image.js
+ * Component Name: Image
+ * Project: design-system
+ * Date: Sat 09/12/2023 - 11:19
+ */
+
+import { IImageComponentProps } from './image.interface';
+import { EImageLoadingType } from './image.enum';
+import { baseImageGeneral } from './image-css';
+
+const Image = ({
+  src,
+  lazy = EImageLoadingType.EAGER,
```

Comment:
Perhaps you could rename `lazy `prop to `loadingType`.  It can be confusing, especially when the default value is `eager`.

Reply:
good Point. I will update it